dspace-api/src/test/data/dspaceFolder/config/local.cfg

```diff
@@ -174,3 +174,6 @@ authority.controlled.dspace.object.owner = true
 # Configuration required for thorough testing of browse links
 webui.browse.link.1 = author:dc.contributor.*
 webui.browse.link.2 = subject:dc.subject.*
+
+# Enable duplicate detection for tests
+duplicate.enable = true
```

Comment:
NOTE: I realized this *might be necessary* if you use the `@ConditionalOnProperty` approach to enable/disable your controller.  As I think that does require you enable the controller in the global `local.cfg` for the Test Environment...since the controller decides whether to load when the application boots up.

Reply:
As per your other comment, i tested and we do need this with the ConditionalProperty annotation used in our controller